### PR TITLE
Enable test checking in Mergify

### PR DESCRIPTION
**Pull request by jpichon**

### CODE REVIEW

**Comment by jpichon** [comment]
Testing with Mergify config editor on this PR and the checks output appears to get picked up correctly.

**Review by jguiditta — APPROVED** [review]

File: .mergify.yml [hunk 1]
```diff
@@ -3,33 +3,30 @@
 queue_rules:
   - name: default
     conditions:
-      - '#approved-reviews-by>=2'
```
Comment (fuzzball81) [inline .mergify.yml:6]: Do we want to drop this?

**Review by fuzzball81 — COMMENTED** [review]
LGTM, one small comment.

File: .mergify.yml (hunk 1 above)
Comment (jpichon) [reply .mergify.yml:6]: We don't have it in the other projects, I only included it in the original version so the conditions wouldn't be empty. The important place is to have it in the merge_requests_rules below.

**Review by fuzzball81 — APPROVED** [review]

**Pull request merged**